src/main/resources/cli/cliInit.js

```diff
@@ -19,16 +19,22 @@ function runInit(runInfo) {
              directory.mkdir()
          }
     } else {
-    	installDir = new File(".").getAbsolutePath()
+    	installDir = new File("").getAbsolutePath()
     }
 	
-	var fileCount = Files.list(Paths.get(installDir)).count()
+	var installDirFile = new File(installDir)
+	
+	var fileCount = Files.list(installDirFile.toPath()).count()
 	
 	if (fileCount > 0) {
-		print('[ERROR] It seems that you already have files on \'' + installDir + '\'. We need a new/clean directory.')
-		return
+		if (runInfo.options.force) {
+			FileUtils.cleanDirectory(installDirFile);
+		} else {
+			print('[ERROR] It seems that you already have files on \'' + installDir + '\'. We need a new/clean directory.')
```

Comment:
Sugiro alterar a mensagem para algo parecido com "The directory + installDir + must be empty. You can use -f option to force ..."